### PR TITLE
news.biglobe.ne.jp

### PR DESCRIPTION
**Pull request by gork7777**
issue: #103126

### CODE REVIEW

**Comment by Sergey-Lyapin** [comment]
@gork7777 please block ads on the main page `biglobe.ne.jp` too

File: AnnoyancesFilter/sections/self-promo.txt [hunk 1]
```diff
@@ -50,6 +50,8 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/103126
+biglobe.ne.jp##.prom
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/self-promo.txt:54]: a lot of these elements are their own ads or another useful links

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -31,6 +31,10 @@ news24.jp,michill.jp,fnn.jp,bengo4.com,zigsow.jp,navitime.com,cazual.shufu.co.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+||top.bcdn.jp/i/saiyou/200424_saiyou.png
+biglobe.ne.jp###join-bnr
+biglobe.ne.jp###spage-r
```
Comment (Sergey-Lyapin) [inline JapaneseFilter/sections/specific.txt:36]: I think we should not block this element

File: JapaneseFilter/sections/specific.txt [hunk 3]
```diff
@@ -31,6 +31,10 @@ news24.jp,michill.jp,fnn.jp,bengo4.com,zigsow.jp,navitime.com,cazual.shufu.co.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+||top.bcdn.jp/i/saiyou/200424_saiyou.png
```
Comment (Sergey-Lyapin) [inline JapaneseFilter/sections/specific.txt:34]: I think we should not block this element

**Review by Sergey-Lyapin — COMMENTED** [review]
@Yuki2718 could you please review this PR too?

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt:54]: On my end this rule has no hit.

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:36]: Maybe okay for Annoyances, but I'd block `###spage-bnr` than `###spage-r` as the former is more annoying (these two are the same).

File: JapaneseFilter/sections/specific.txt (hunk 3 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:34]: Agreed

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/self-promo.txt:54]: @gork7777 please remove this rule

File: JapaneseFilter/sections/specific.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply JapaneseFilter/sections/specific.txt:34]: @gork7777 please remove this rule

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply JapaneseFilter/sections/specific.txt:36]: Please move to annoyances.txt with a comment

**Comment by Yuki2718** [comment]
There are some staff I missed at the initial review, will fix directly.

**Pull request merged**